ftplugin/java.lua

```diff
@@ -20,6 +20,54 @@ elseif vim.fn.has("unix") == 1 then
 else
 	print("Unsupported system")
 end
+local function set_keymaps(buffer)
```

Comment:
please use this simplified function instead

```lua
  local nmap = function(lhs, rhs, desc)
    if desc then
      desc = desc
    end

    vim.keymap.set("n", lhs, rhs, { silent = true, desc = desc })
  end
```

then simply do

```lua
local jtdls = require('jdtls')
nmap("<leader>Jo", jdtls.organize_imports, "organize imports")
nmap("<leader>Jv", jdtls.extract_variable, "extract variable")
-- ...
```

Reply:
@ChristianChiarulli what do you say? Perhaps this should be done for all branches? Currently the format is which-key format, which I think is fine. WDYT @kylo252?

Reply:
We'd really appreciate it if you could manage to fix all the branches, that would be really helpful!!

Reply:
I'll see what I can do. :)